docs/guides/elasticsearch/x-pack/connect-kibana.md

```diff
@@ -0,0 +1,571 @@
+statefulset.apps/es-kibana-demo   1         1         2m
+```
+
+Once everything is created, Elasticsearch will go to Running state. Check that Elasticsearch is in running state.
+
+```console
+$ kubectl get es -n demo es-kibana-demo
+NAME             VERSION   STATUS    AGE
+es-kibana-demo   6.3.0     Running   5m
+```
+
+Now, check elasticsearch log to see if the cluster is ready to accept requests,
+
+```console
+$ kubectl logs -n demo es-kibana-demo-0 -f
+...
+Starting runit...
+...
+Elasticsearch Version: 6.3.0
+Search Guard Version: 6.3.0-23.0
+Connected as CN=sgadmin,O=Elasticsearch Operator
+Contacting elasticsearch cluster 'elasticsearch' and wait for YELLOW clusterstate ...
+Clustername: es-kibana-demo
+Clusterstate: GREEN
+Number of nodes: 1
+Number of data nodes: 1
+...
+Done with success
+...
+```
+
+Once you see `Done with success` success line in the log, the cluster is ready to accept requests. Now, it is time to connect with Kibana.
+
+## Deploy Kibana
+
+In order to connect the Elasticsearch cluster that we have just deployed, we need to configure `kibana.yml` with appropriate configuration.
+
+First, we need to know the IP address of the service created by KubeDB for `es-kibana-demo` cluster. Let's retrieve by,
+
+```console
+$ kubectl get service -n demo es-kibana-demo
+NAME             TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)    AGE
+es-kibana-demo   ClusterIP   10.102.246.114   <none>        9200/TCP   6m
+```
+
+Here, `10.102.246.114` is our desired IP address. Now, configure `kibana.yml` as below,
+
+```yaml
+xpack.security.enabled: false
+server.host: 0.0.0.0
+
+elasticsearch.url: "http://10.102.246.114:9200"
+elasticsearch.username: "kibanauser"
+elasticsearch.password: "kibana@secret"
+
+searchguard.auth.type: "basicauth"
+searchguard.cookie.secure: false
+```
+
+Notice the `elasticsearch.username` and `elasticsearch.password` field. Kibana will connect to Elasticsearch cluster with this credentials. They must match with the credentials we have provided in `sg_internal_users.yml` file while creating the cluster.
+
+Now, create a ConfigMap with `kibana.yml` file. We will mount this ConfigMap in Kibana deployment so that Kibana starts with this configuration.
+
+```conlose
+$ kubectl create configmap -n demo kibana-config \
+                        --from-file=./kibana.yml
+configmap/kibana-config created
+```
+
+Finally, deploy Kibana deployment,
+```console
+$ kubectl apply -f https://raw.githubusercontent.com/kubedb/cli/0.8.0/docs/examples/elasticsearch/x-pack/kibana-deployment.yaml
+deployment.apps/kibana created
+```
+
+Below is the YAML for the Kibana deployment we just created.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kibana
+  namespace: demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kibana
+  template:
+    metadata:
+      labels:
+        app: kibana
+    spec:
+      containers:
+      - name: kibana
+        image: kubedbci/kibana:6.3.0
```

Comment:
Add the Dockerfiles to hack/docker/kibana and push to kubedb docker hub account

Reply:
added